✨ The 19th community highlight! (#17649)
home-assistant/home-assistant.github.io: source/_posts/2021-04-30-community-highlights.markdown

date: 2021-04-30 00:00:00
date_formatted: "April 30, 2021"
author: Klaas Schoute
author_twitter: klaasnicolaas
categories: Community
og_image: /images/blog/2021-04-30-community-highlights/social.png
---

The 19th edition of the Home Assistant Community Highlights! Some interesting
things popped up around our community, we thought was worth sharing.

Amazing to see how many reactions there have been after sharing the
[internship progress][internship], we have seen many cool things and if you have
any input you can still share this of course! ❤️

Do you want to share something for the next edition?
Information on [how to share](#got-a-tip-for-the-next-edition).

./Klaas

## Blueprint of the week
------

<div style="margin:0 auto; text-align:center">
    <a href="https://community.home-assistant.io/t/vacuum-schedule/256590" target="_blank">
        <img
            src='/images/blog/2021-04-30-community-highlights/blueprint.png'
            alt="Setting up the Blueprint"
            style='border: 0;box-shadow: none;width:70%;margin-bottom:15px;'
        />
    </a>
</div>

> This week's blueprint is that of [delphiki](https://community.home-assistant.io/u/delphiki),
> with which you can plan on which days of the week your vacuum cleaner should start
> cleaning and at what time. Try it out! Read more about it on the [community forum][week_blueprint]
> or install this automation in your instance with a click on the my button!

Reviewer: ```suggestion
This week's blueprint is that of [delphiki](https://community.home-assistant.io/u/delphiki),
with which you can plan on which days of the week your vacuum cleaner should start
cleaning and at what time. Try it out! Read more about it on the [community forum][week_blueprint]
or install this automation in your instance with a click on the My button!
```

        <img
            src='/images/blog/2021-04-30-community-highlights/vacuum.png'
            alt="The vacuum card"
            style='border: 0;box-shadow: none;width:70%;margin-bottom:15px;'
        />
    </a>
</div>

Besides being able to automate when your vacuum cleaning robot gets started,
you probably also want to show the status of your robot in Lovelace. Then try
the [vacuum card](https://github.com/denysdovhan/vacuum-card) from [denysdovhan](https://github.com/denysdovhan),
with which you can show that in a beautiful way and there is support for the
well-known models that are now available on the market.

<a href="https://github.com/denysdovhan/vacuum-card"><img style="border: 0;" src="https://gh-card.dev/repos/denysdovhan/vacuum-card.svg"></a>

## OpenTTS 2.0
------

Maybe the name still sounds fairly unknown to you, but [OpenTTS](https://github.com/synesthesiam/hassio-addons)
is an add-on, which gives you the possibility to use multiple open source
Text to Speech systems. So that you can eventually have text spoken on: for
example, a Google Home speaker. [synesthesiam](https://github.com/synesthesiam)

> recently released a new release for OpenTTS and you can install it as an

Reviewer: ```suggestion
recently released a new version of OpenTTS and you can install it as an
```
